### PR TITLE
Allow external Wi-Fi adapter on Toyota HSR

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
As discussed in both #502 and #533.

The conclusion of these issues are to open a PR and allow the TC to review it. Hereby my suggested rules regarding an external Wi-Fi adapter on the Toyota HSR in DPSL.

Update 22-05-2019:
I reopened this PR based on our experiences in the German Open. See #502 for the discussion.

### CODE REVIEW

File: general_rules/ExternalDevices.tex [hunk 1]
```diff
@@ -59,10 +59,16 @@ \subsubsection{Technical Specifications}
...
+The referees, Technical Committee, and Organizing Committee members may run random checks anytime during the competition prior to the test to verify that the laptop in the Toyota HSR \iterm{Mounting Bracket} has no additional hardware plugged in, and matches the authorized specifications.
 
+\subsubsection{External Wi-Fi adapter}
+As described in section 6.2.2.5.6 of the Toyota HSR manual, an external Wi-Fi adapter can be used to increase stability and/or bandwidth of the network connection. As teams are allowed to use the network adapter of the \iterm{OSL}, teams may also chose to use an external Wi-Fi adapter.
+
+No constraints are applied regards the Wi-Fi specifications. However to be used on the Toyota HSR, the Wi-Fi adapter should be USB-powered. To allow wired network connections between the internal computer, the \iterm{OSL} and the external Wi-Fi adapter, a (USB-powered) switch is allowed.
```
Comment (johaq) [inline general_rules/ExternalDevices.tex]: "in regards to the Wi-Fi" or "regarding the Wi-Fi"
And it should be "adapter specifications.", right?
Is there enough space in the backback for the standard laptop and a switch?
Or more general do we still require all this additional equipment to fit inside the backback?

Comment (MatthijsBurgh) [reply general_rules/ExternalDevices.tex]: Fix the grammar issue.

A switch and Wi-Fi adapter don't fit in the backpack. We taped them on side of the robot. As the base is bigger than the torso. These devices don't or just barely increase the size of the robot.

**Comment by kyordhel** [comment]
@MatthijsBurgh please check #635

**Comment by kyordhel** [comment]
Solved in #635

**Pull request closed without merging**